EffortlessUI-new/public/base.css

```diff
@@ -0,0 +1,384 @@
+  --lightgreen: #42b72a;
+  --purple: #7952b3;
+  --lightOrange: #ff9f00;
+  --orange: #fb641b;
+  --greenishBlue: #17a2b8;
+  --grey: #f0e9e6;
+  --maroon: #cd486b;
+}
+
+/* Buttons */
+/* Primary Button */
+.primary-button {
+  cursor: pointer;
+  padding: 0.8rem 2rem;
+  color: white;
+  font-size: medium;
+  box-shadow: 0px 2px 4px 1px rgb(0 0 0 / 20%);
+  border: none;
+  border-radius: 0.2rem;
+}
+.primary-button.purple {
+  background-color: var(--purple);
+}
+.primary-button.greenishBlue {
+  background-color: var(--greenishBlue);
+}
+.primary-button.blue {
+  background-color: var(--blue);
+}
+.primary-button.pink {
+  background-color: var(--pink);
+}
+.primary-button.red {
+  background-color: var(--red);
+}
+.primary-button.lightOrange {
+  background-color: var(--lightOrange);
+}
+.primary-button.yellow {
+  background-color: var(--yellow);
+}
+.primary-button.lightgreen {
+  background-color: var(--lightgreen);
+}
+.primary-button.orange {
+  background-color: var(--orange);
+}
+.primary-button.grey {
+  background-color: var(--grey);
+}
+.primary-button.maroon {
+  background-color: var(--maroon);
+}
+
+/* icon button */
+.icon-button {
+  cursor: pointer;
+  background-color: var(--grey);
+  border: none;
+  padding: 0.5rem;
+}
+
+.icon-button > img {
+  height: 30px;
+  width: 30px;
+}
+
+.transparentBtn {
+  cursor: pointer;
+  padding: 0.2rem 1rem;
+  background-color: white;
+  color: black;
+  border: 2px solid var(--grey);
+  outline-style: none;
+}
+
+/* social media icon */
+.socialIcon > img {
+  height: 3rem;
+  width: 3rem;
+}
+
+/* Avatar */
+.faceavatar {
+  height: 50px;
+  width: 50px;
+  border-radius: 50%;
+}
+
+.avatar-alphabet {
+  width: 3rem;
+  height: 3rem;
+  color: white;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+  font-size: 2rem;
+  align-content: center;
+  font-weight: 900;
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+/* alert */
+.alert {
+  display: flex;
+  align-items: center;
+}
+
+.alertText {
+  background-color: white;
+  border: 1px solid black;
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+  width: 20rem;
+  display: flex;
+  justify-content: space-between;
+  padding: 1rem 2rem 1rem 0.5rem;
+  align-items: center;
+}
+
+.alertText.success {
+  color: green;
+}
+.alertText.failure {
+  color: red;
+}
+.alertText.warning {
+  color: #d97706;
+}
+.alertText.info {
+  color: var(--blue);
+}
+
+.alertIcon {
+  height: 42px;
+  padding: 0.3rem 0.5rem;
+}
+
+.alertIcon.success {
+  background-color: #42b72a;
+}
+.alertIcon.failure {
+  background-color: rgb(179, 11, 11);
+}
+.alertIcon.warning {
+  background-color: #d97706;
+}
+.alertIcon.info {
+  background-color: var(--blue);
+}
+
+.alert > div > img {
+  width: 40px;
+  height: 40px;
+}
+
+.cross {
+  cursor: pointer;
+}
+
+@media screen and (max-width: 520px) {
+  .alertText {
+    width: 14rem;
+  }
+}
+
+/* Navbar */
+.nav1__navList {
```

Comment:
Great that you are following BEM convention here, Just a suggestion that you can reflect this everywhere. It looks good.